src/Leanplum.js

```diff
@@ -136,6 +136,30 @@ export default class Leanplum {
     InternalState.removeVariablesChangedHandler(handler)
   }
 
+  static forceContentUpdate(callback) {
+    LeanplumRequest.request(Constants.METHODS.GET_VARS,
+      new ArgsBuilder()
+      .add(Constants.PARAMS.INCLUDE_DEFAULTS, false), {
+        queued: false,
+        sendNow: true,
+        response: function (response) {
+          let getVarsResponse = LeanplumRequest.getLastResponse(response);
+          let success = false;
+          if (LeanplumRequest.isResponseSuccess(getVarsResponse)) {
+            let values = getVarsResponse[Constants.KEYS.VARS];
+            let variants = getVarsResponse[Constants.KEYS.VARIANTS];
+            let actionMetadata = getVarsResponse[Constants.KEYS.ACTION_METADATA];
+            VarCache.applyDiffs(values, variants, actionMetadata)
```

Comment:
Let's use the same format as the start's response.
VarCache.applyDiffs(
               startResponse[Constants.KEYS.VARS],
               startResponse[Constants.KEYS.VARIANTS],
               startResponse[Constants.KEYS.ACTION_METADATA])